src/App.js

```diff
@@ -26,34 +32,55 @@ class App extends React.Component {
 
   handleClick = async (e) => {
       e.preventDefault();
-
+      try{
       const locationAPI = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_LOCATION_API}&q=${this.state.searchQuery}&format=json`;
       const locationResponse = await axios.get(locationAPI);
-      console.log(locationResponse);
       this.setState({
         lat: locationResponse.data[0].lat,
         lon: locationResponse.data[0].lon,
         display_name: locationResponse.data[0].display_name
       });
+
+      const mapAPI =`https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATION_API}&lat=${this.state.lat}&lon=${this.state.lon}&center=${this.state.lat},${this.state.lon}&zoom=10&theme=streets`;
+      const mapResponse = await axios.get(mapAPI);
+      this.setState({
+        map: mapResponse.config.url
```

Comment:
I didn't think to do this.  I like how you targeted info in the response rather than doing what i did and pulling the base link off of location IQ and filling it in with template literals.